docs/data/toolpad/core/components/dashboard-layout/dashboard-layout.md

```diff
@@ -22,9 +22,9 @@ The `navigation` prop in the `AppProvider` allows for setting any type of naviga
 
 {{"demo": "DashboardLayoutNavigation.js", "height": 640, "iframe": true}}
 
-## API
+<!-- ## API
```

Comment:
Let's leave API docs generation to another PR as it doesn't seem super simple and we can discuss how to do it.